src/shared/utils/download.ts

```diff
@@ -1,15 +1,6 @@
 export const download = (filename: string, mediaType: string, data: any) => {
   const blob = new Blob([data], { type: mediaType });
-  if (window.navigator.msSaveOrOpenBlob) {
-    window.navigator.msSaveBlob(blob, filename);
-  } else {
-    const elem = window.document.createElement('a');
-    elem.href = window.URL.createObjectURL(blob);
-    elem.download = filename;
-    document.body.appendChild(elem);
-    elem.click();
-    document.body.removeChild(elem);
-  }
+  window.navigator.msSaveBlob(blob, filename);
```

Comment:
Why did we remove the else block?

Reply:
that `window.navigator.msSaveOrOpenBlob` function is guaranteed to exist